src/domain/entities/Log.ts

```diff
@@ -27,26 +27,20 @@ export type LogAction =
 export type LogStatus = "success" | "failed";
 export type PartialLog = Pick<Log, "action" | "status">;
 
+const ACTION_DESCRIPTIONS: Record<LogAction, string> = {
+    sharing: i18n.t("change sharing settings"),
+    orgunits: i18n.t("change organisation units"),
+    delete: i18n.t("delete"),
+    edit: i18n.t("edit dataset"),
+    create: i18n.t("create new dataset"),
+    clone: i18n.t("clone dataset"),
+    period_dates: i18n.t("change period dates"),
+    unknown: "",
+};
```

Comment:
i18n.t cannot be directly used at root-level, converting it to an arrow function should be enough